esp/templates/users/userchecklist.html

```diff
@@ -14,6 +14,19 @@
 <br />
 <br />
 
+<script type="text/javascript">
+    function checkAll(checkId){
+        var inputs = document.querySelectorAll("checkId");
```

Comment:
I think you want `checkId` here -- the variable, not the string.  Also, I think you'll need to prefix this with a `.`, so you get things with the _class_ `checkId`, not the node type.  See [querySelectorAll docs here](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll).

Reply:
It seems to function either way-- but I could have just been lucky before!